internal/engine/buildcontroller_test.go

```diff
@@ -151,14 +151,14 @@ func TestBuildControllerIgnoresImageTags(t *testing.T) {
 	f := newTestFixture(t)
 	defer f.TearDown()
 
-	ref := container.MustParseNamed("image-foo:tagged")
+	ref := container.MustParseNamed("gcr.io/blorg-dev/blorg-backend:devel-nick")
 	refSel := container.NewRefSelector(ref)
 
 	iTarget := NewSanchoLiveUpdateImageTarget(f)
 	iTarget = iTarget.MustWithRef(refSel)
 
 	manifest := manifestbuilder.New(f, "fe").
-		WithK8sYAML(SanchoYAML).
+		WithK8sYAML(testyaml.BlorgJobYAML).
```

Comment:
The image targets need to actually exist in the YAML, and `image-foo:tagged` does not exist in Sancho. The Blorg Job uses a tag (which is critical to this test), so was ideal to borrow here.